ignite/contrib/metrics/MARE.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import division
+from ignite.exceptions import NotComputableError
+from ignite.metrics import Metric
+import torch
+
+
+class MeanAbsoluteRelativeError(Metric):
+    """
+    Calculate Mean Absolute Relative Error
+
```

Comment:
In case of #307 maybe we can put some metrics mathematical definitions and link to the article https://arxiv.org/ftp/arxiv/papers/1809/1809.03006.pdf

Reply:
That's good idea. I can add equations and link to the paper.

Reply:
@vfdev-5 It's done. Let me know if there is more modifications needed.